### ISSUE TITLE
[BUG] Performance issue in PTMC Research Bunker when Render depth is set higher than 40 (minimum value)

### ISSUE THREAD

**Issue by Sol1vaN** [InsanityBringer/PiccuEngine#57]
It's seems a vanilla problem, it's explain why the Render depth control reset to minimum when you load that map.
May Interplay devs noticed this but they don't have time to resolve it, only with this ugly trick reseting the render depth control to 40.

I've tested every piccuengine and this happens in all cases.
For other hand, with vanilla versions the FPS works fine, because, again, the render depth are set to 40 automatically by the game when you enter to PTMC Research Bunker

Another test that I've done is trough piccuengine set render depth to 40, and the FPS works fine, but the map look bad, with those violent clipping in the near distance.

**Comment by InsanityBringer** [comment]
yeah this is a known problem, if the terrain seg limit ever gets hit it resets your render depth, and will be fixed with Newrender. 

In the meantime turn down your terrain detail a notch or two. it sucks but the original rendering code sucks
